### PR TITLE
Adjusting apicast resources requirements [THREESCALE-598]

### PR DESCRIPTION
**Pull request by maneta**
Motivated by the insertion of Apicast template in OSO.

@jmprusi @mikz Do you think that those values are good for everyone or do you think that maybe we should have a separated template just for Openshift Online Starter version.

Closes: https://issues.jboss.org/browse/THREESCALE-598

### CODE REVIEW

File: apicast-gateway/apicast.yml [hunk 1]
```diff
@@ -82,6 +82,13 @@ objects:
           - name: management
             containerPort: 8090
             protocol: TCP
+          resources:
+          limits:
+              cpu: '1'
+              memory: 512Mi
```
Comment (mikz) [inline apicast-gateway/apicast.yml]: I think it could easily live with a half of that. Even with 128M.

Comment (jmprusi) [reply apicast-gateway/apicast.yml]: 128 ~150Mb , depends on the number of apicast workers.. 

And I don't see any change to the default APICAST_WORKERS...

Comment (maneta) [reply apicast-gateway/apicast.yml]: @jmprusi can you review this PR? 

We have the APICAST_WORKERS in place now and QE have already tested it

File: apicast-gateway/apicast.yml [hunk 2]
```diff
@@ -105,7 +114,7 @@ objects:
 parameters:
 - name: AMP_RELEASE
   description: "AMP release tag."
-  value: 2.1.0-CR2-redhat-1
+  value: "2.0.0.GA"
```
Comment (jmprusi) [inline apicast-gateway/apicast.yml]: Shouldn't this be 2.0.1.GA ?

File: apicast-gateway/apicast.yml [hunk 3]
```diff
@@ -119,6 +128,10 @@ parameters:
   value: production
   name: DEPLOYMENT_ENVIRONMENT
   required: true
+- description: "Number of worker processes to start."
```
Comment (jmprusi) [inline apicast-gateway/apicast.yml]: I think we can remove this

File: apicast-gateway/apicast.yml (hunk 2 above)
Comment (maneta) [reply apicast-gateway/apicast.yml]: done in dedce9bb0089fb28c3993c590908fa07e885c47a

File: apicast-gateway/apicast.yml (hunk 3 above)
Comment (maneta) [reply apicast-gateway/apicast.yml]: done in dedce9bb0089fb28c3993c590908fa07e885c47a

**Pull request merged**